test/jetstream-e2e/testdata/nats-auth-fake-token.yaml

```diff
@@ -0,0 +1,9 @@
+
+apiVersion: v1
```

Comment:
Where is this file used?

Reply:
it is getting used in the following line: /go:generate kubectl apply -f testdata/nats-auth-fake-token.yaml -n numaflow-system